server/src/main/java/com/cloud/event/Taxonomies.java

```diff
@@ -0,0 +1,451 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package com.cloud.event;
+
+import java.util.HashMap;
+import java.util.UUID;
+
+public class Taxonomies {
+    //eventMapping maps CloudStack Event Category (substring of EventType) to CADF Target Resource
+    public static HashMap<String, String> cstoCadfResourceMapping = new HashMap<String, String>();
+    public static HashMap<Taxonomies.Action, Taxonomies.EventType> eventActionToTypeMapping = new HashMap<Taxonomies.Action,
+            Taxonomies.EventType>();
+    public static HashMap<String, String> eventResourcetoUuidMapping = new HashMap<String, String>();
+
+
+
+    public Taxonomies() {
```

Comment:
Taxonomies could be a sub-package? It has no instance fields or methods exept for and only this constructor. All 'member' enums could be in their own files.

I see no point in this public constructor.